src/server/services/user/createUser.js

```diff
@@ -37,7 +37,7 @@ const { getConnection } = require('../../db');
 	}
 
 	const passwordHash = bcrypt.hashSync(password, 10);
-	const admin = new User(undefined, email, passwordHash, User.role.ADMIN);
+	const admin = new User(undefined, email, passwordHash, User.role.ADMIN, undefined);
```

Comment:
In maps, it saves the calibration time with ``modifiedDate: moment().toISOString()``. OED also puts a message about automatically created units, etc. in the note. How about we put: `` `User ${email} created via createUser on ${moment().toISOString()}` `` as the note.

Reply:
Done.